### PR TITLE
docs: Update instructions for adding DEB repository

### PR DESCRIPTION
**Pull request by wido**
apt-key and apt-get are deprecated. This PR makes use of the proper commands

### CODE REVIEW

File: source/installguide/management-server/_pkg_repo.rst [hunk 1]
```diff
@@ -99,19 +99,19 @@ repository to the file (replace "trusty" with "xenial" or "bionic" if it is the
...
 
-   deb http://download.cloudstack.org/ubuntu focal |version|
+   deb https://download.cloudstack.org/ubuntu focal |version|
 
 We now have to add the public key to the trusted keys.
 
 .. parsed-literal::
 
-   wget -O - http://download.cloudstack.org/release.asc |sudo apt-key add -
+   wget -O - https://download.cloudstack.org/release.asc |sudo tee /etc/apt/trusted.gpg.d/cloudstack.asc
```
Comment (rohityadavcloud) [inline source/installguide/management-server/_pkg_repo.rst:108]: Should this be, something like:
```
mkdir -p /etc/apt/keyrings
wget -O- https://download.cloudstack.org/release.asc | gpg --dearmor | sudo tee /etc/apt/keyrings/cloudstack.gpg > /dev/null
```
And for use for the repo, the usage is like or something similar:
```
echo deb [signed-by=/etc/apt/keyrings/cloudstack.gpg] https://download.cloudstack.org/<path to repo here> / > /etc/apt/sources.list.d/cloudstack.list
```

**Review by DaanHoogland — APPROVED** [review]

File: source/installguide/management-server/_pkg_repo.rst [hunk 2]
```diff
@@ -99,19 +99,19 @@ repository to the file (replace "trusty" with "xenial" or "bionic" if it is the
...
 .. parsed-literal::
 
-   deb http://download.cloudstack.org/ubuntu focal |version|
+   deb https://download.cloudstack.org/ubuntu focal |version|
 
 We now have to add the public key to the trusted keys.
 
 .. parsed-literal::
 
-   wget -O - http://download.cloudstack.org/release.asc |sudo apt-key add -
```
[the remark this answers is not in the document]
Comment (wido) [reply source/installguide/management-server/_pkg_repo.rst:108]: I have tested this and it worked on the Ubuntu 22.04 systems I tested this on

File: source/installguide/management-server/_pkg_repo.rst (hunk 1 above)
Comment (wido) [reply source/installguide/management-server/_pkg_repo.rst:108]: Would that really be needed that way? I have tested the commands and worked on my environment.

Happy to change it though

**Comment by DaanHoogland** [comment]
one more thing @wido do we need this on the 4.18 branch?

**Review by weizhouapache — APPROVED** [review]
code lgtm

**Pull request merged**